Rethinking_2/Chp_07.ipynb

```diff
@@ -33,7 +33,7 @@
    "source": [
```

Comment:
<p>Thanks for the suggestion, but I think it's better to stay with the ArviZ way here, for two reasons:</p><ol><li>To show people that you can do that with ArviZ and stay in the "ArviZ ecosystem"</li><li>To be able to compare the output of this summary with the summary of Bayesian model later -- it's nice to get the HDI and ESS ;)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Hi Alex,</p><p> </p><p>I agree that displaying hpd and ess is intertesting. However, usage of arviz for this case generates a nasty warning that seems to suggest that the usage of API for this shape of data is invalid. az.summary expects a minimum shape of (chains=2, draws=4) which is not what is passed. </p><p> </p><p>Don't you think this would leave the (new) user who is not familiar with xarray and inferencedata bit puzzled ? In any s/w application, if warnings are not attended to they become the norm. </p><p> </p><p>This is the main reason I replaced. </p><p> </p><p>I have put it back with one tiny change. i.e. addition of parameter kind="stats". It removes the warning. You still have hdi displayed but no ess</p><p> </p><p>az.summary({k: v for k, v in zip("ba", post.T)}, kind="stats")</p><p> </p><p>Hope this is acceptable</p><p> </p>

Reply:
Ok, let's go with that, sounds good enough 😉